protocol_tests/connection/__init__.py

```diff
@@ -30,8 +31,8 @@ class DIDDoc(dict):
     # DIDDoc specification is very flexible: https://w3c-ccg.github.io
     # This particular schema covers Ed25519 keys. All key types here:
     # https://w3c-ccg.github.io/ld-cryptosuite-registry/
-    EXPECTED_SERVICE_TYPE = 'IndyAgent'
```

Comment:
This conflicts with updated language in https://github.com/hyperledger/aries-rfcs/blob/master/features/0160-connection-protocol/README.md#diddoc-example